src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1656,6 +1656,18 @@ components:
     # Common Response Types
     #
 
+    TransactionType:
```

Comment:
I think we're calling it TransactionClassification or TransactionCategory rather than "type"

Reply:
i did go for `ManifestClass` which is how Omar called that in toolkit and I think it works better. What do you think?